tests/bdd/steps/feature_steps/lms.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Steps for our app
+  Scenario: the assigment opens successfully
+    Then the response is HTML
+    Then the response status code is 200
+
+  Scenario: the user has instructor privileges
```

Comment:
Now are we talking about an LMS-app user?

Reply:
The real person again.